src/main/java/com/ericsson/eiffel/remrem/semantics/eiffelSchemas/SchemaFile.java

```diff
@@ -0,0 +1,221 @@
+package com.ericsson.eiffel.remrem.semantics.eiffelSchemas;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.Map.Entry;
+import java.util.Set;
+import org.apache.commons.lang3.StringUtils;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+
+/**
+ * 
+ * This class is used to add the required properties to generate the pojo's
+ * using jsonSchema2pojo plugin.
+ * 
+ * @author xdurvak
+ *
+ */
+public class SchemaFile {
+	private static String fileContent;
+	private final static JsonParser parser = new JsonParser();
+	private static int count = 0;
+	static boolean isItemObject = false;
+	private static String oldName = "";
```

Comment:
I think this should not be a class variable. Move declaration to where it is set add to method declaration for addingItemsProperties